apps/sms/views/shared/js/settings.js

```diff
@@ -35,14 +36,16 @@ var Settings = {
     var keyHandlerSet = {
       'dom.mms.operatorSizeLimitation': this.initMmsSizeLimitation.bind(this),
       'operatorResource.sms.maxConcat':
-        this.initSmsMaxConcatenatedMsg.bind(this)
+        this.initSmsMaxConcatenatedMsg.bind(this),
+      'messages.mms.sendReadReport.enabled':
```

Comment:
@julienw  Why i am getting this error: Reference error : Settings is not defined
When i replaced the string with this.SEND_READ_REPORT_KEY